terraform/fluentd/k8s/containers.conf

```diff
@@ -77,14 +77,14 @@
 </label>
 
 <label @rack>
-	# <filter **>
-	# 	@type concat
-	# 	key log
-	# 	multiline_start_regexp /^\S/
-	# 	separator ""
-	# 	flush_interval 5
-	# 	timeout_label @target
-	# </filter>
+    <filter **>
+      @type record_modifier
+      <record>
+        type  k8s
+        log ${record["message"]}
+      </record>
+      remove_keys message
+    </filter>
```

Comment:
why did you uncomment this?

Reply:
i didn't uncomment it, i removed that portion and added filter for cri changes
github diff showing it differently